Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8-stretch
+FROM node:12.18.1-stretch
```

Comment:
```suggestion
FROM node:12.18.1-stretch
```

Thanks! I'd rather `node:12-stretch` instead; node is stable enough to not have to pin a specific precise version.

Reply:
Hi,
Thanks for the suggestion, but you did not downgrade the version in the file of your suggested change so I have updated the file to `node:12-stretch` in my pull request.